src/pudl/io_managers.py

```diff
@@ -695,37 +695,23 @@ def filter_for_freshest_data(
         Each row in our SQLite database includes all the facts for one context/filing
         pair.
 
-        If one context is represented in multiple filings, we take the facts from the
-        most recently-published filing.
+        If one context is represented in multiple filings, we take the most
+        recently-reported non-null value.
 
-        This means that if a recently-published filing does not include a value for a
-        fact that was previously reported, then that value will remain null. We do not
-        forward-fill facts on a fact-by-fact basis.
+        This means that if a utility reports a non-null value, then later
+        either reports a null value for it or simply omits it from the report,
+        we keep the old non-null value, which may be erroneous. This appears to
+        be fairly rare, affecting < 0.005% of reported values.
```

Comment:
can we add a little defensive check in the here to make sure that expected % of values stays semi-constant?

Reply:
Hm, this is a little tricky - we'd have to calculate both the `best snapshot` and the `apply diffs` methodologies, stack them all up, then compare the two - which we *can* do, but seems like a lot of complexity to add for this check.

Reply:
Yea i didn't realize how complicated this request was going to be. I don't think what you have here is too too complicated but it is a lot more than was here before. I'd be inclined to keep it in but if you feel otherwise I think it would be fine to revert to not checking this.